sklearn/neighbors/base.py

```diff
@@ -141,7 +141,9 @@ def _check_algorithm_metric(self):
             if self.algorithm == 'kd_tree':
                 # callable metric is only valid for brute force and ball_tree
                 raise ValueError(
-                    "kd_tree algorithm does not support callable metric '%s'"
+                    "kd_tree does not support callable metrics.'%s'"
```

Comment:
```suggestion
                    "kd_tree does not support callable metric '%s'"
```

Reply:
I think the current version "does not support callable metrics." is fine. Same as you would say "support callable functions"  not "function".

Reply:
the issue is that we get open issues because people want to pass a callable and they can't. Explaining why we don't support it, helps them understand and go another path.

Reply:
You are right, never-mind, I missed the string formatting at the end.